src/NUnit3TestAdapterInstall/NUnit3TestAdapterInstall.csproj

```diff
@@ -27,27 +19,11 @@
     <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
     <DeployExtension>False</DeployExtension>
     <IsProductComponent>false</IsProductComponent>
+    <OutputPath>..\..\bin\$(Configuration)\</OutputPath>
     <VsixInputFileLocation>..\NUnitTestAdapter\bin\$(Configuration)\net35\</VsixInputFileLocation>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>..\..\bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <DeployExtension Condition="'$(AppVeyor)' != ''">True</DeployExtension>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>..\..\bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <DeployExtension Condition="'$(AppVeyor)' != ''">False</DeployExtension>
-  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' " />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' " />
```

Comment:
This is still necessary so that VS can detect which configurations and platforms to show in Configuration Manager. If these conditions are removed, VS changes the solution file to list this project as `x86`.